database/python/2015/3908.yaml

```diff
@@ -0,0 +1,16 @@
+---
+cve: 2015-3908
+title: CVE in ansible
+description: >
+    Ansible before 1.9.2 does not verify that the server hostname matches a domain name in the subject's Common Name (CN) or subjectAltName field of the X.509 certificate, which allows man-in-the-middle attackers to spoof SSL servers via an arbitrary valid certificate.
+cvss_v2: 4.3
+references:
+    - http://lists.opensuse.org/opensuse-updates/2015-07/msg00051.html
+    - http://lists.opensuse.org/opensuse-updates/2015-08/msg00029.html
+    - http://www.ansible.com/security
+    - http://www.openwall.com/lists/oss-security/2015/07/14/4
+
+affected:
+    - name: ansible
+      version:
+        - "<1.9.2"
```

Comment:
Unfortunately this syntax is not supported by VictimsDB: https://github.com/victims/victims-cve-db#version-string-common

You need to use `"<=1.9.1"`

Reply:
@samuzzal-choudhury PTAL :wink: